### ISSUE TITLE
GitHub Actions macOS timeout installing Package Control

### ISSUE THREAD

**Issue by ehuss** [SublimeText/UnitTesting#220]
Ever since GitHub Actions updated the macOS 12 image a few days ago to 20230709.1 (from 20230623.2), the install step for package control has been timing out with the error:

```
Run bash $GITHUB_ACTION_PATH/../../scripts/install_package_control.sh
creating Package Control.sublime-settings
/Users/runner/work/_actions/SublimeText/UnitTesting/v1/actions/setup/../../scripts/install_package_control.sh: line 67:  1618 Terminated: 15          subl
/Users/runner/work/_actions/SublimeText/UnitTesting/v1/actions/setup/../../scripts/install_package_control.sh: line 67:  1840 Terminated: 15          subl
/Users/runner/work/_actions/SublimeText/UnitTesting/v1/actions/setup/../../scripts/install_package_control.sh: line 67:  2055 Terminated: 15          subl
....................................Timeout: Fail to install Package Control.
Error: Process completed with exit code 1.
```

Nothing in the release notes at https://github.com/actions/runner-images/releases/tag/macOS-12%2F20230709.1 seems to jump out as possible culprits.

I've been doing some debugging, and all I can tell is that the `bootstrapped` flag never gets set [here](https://github.com/SublimeText/UnitTesting/blob/be49498d31ed9dac8f0d13479468e85bd0eab36e/sbin/pc_helper.py#L40). 

I'm wondering if you have suggestions on how to better debug the issue? I suspect something useful is being displayed in Sublime's console, but I can't find any APIs or options to programmatically access the console or redirect it to a file. I tried replacing `sys.stderr` and `sys.stdout` inside the `install_package_control_helper.py`, but I didn't get anything out of that.

**Comment by deathaxe** [comment]
Caused by https://github.com/wbond/package_control/issues/1612

**Comment by randy3k** [comment]
I think we could add a flag to support PC 4.0. 
This is a plugin to be injected to Sublime that checks if all the dependencies have been installed. 
https://github.com/SublimeText/UnitTesting/blob/master/scripts/install_package_control_helper.py#L53
If anything may break, it will be the first candidate to check.

**Comment by deathaxe** [comment]
Fixed by #225
